packages/shared/config/getEnvValue.ts

```diff
@@ -2,15 +2,12 @@ export function getEnvValue<T extends string | number>(
   v: string | undefined,
   defaultVal?: T,
 ): T {
-  if (v && process.env[v]) {
-    return typeof defaultVal === 'number'
-      ? (Number(v) as T)
-      : (process.env[v] as T);
+  if (v) {
+    return typeof defaultVal === 'number' ? (Number(v) as T) : (v as T);
   }
 
-  if (defaultVal === undefined)
-    throw new Error(`Missing ENV Variable: ${v} and NO DEFAULT AVAILABLE`);
+  if (defaultVal === undefined) throw new Error('Missing ENV Variable');
 
-  console.warn(`Using default ENV variable: ${defaultVal} for value: ${v}`);
+  console.warn(`Using default ENV variable: ${defaultVal}`);
   return defaultVal;
 }
```

Comment:
@scottrepreneur I had to revert these changes since it wasn't pulling in env variables correctly. probably some weird node thing where its looking specifically for `process.env.foo` to inject the value

Reply:
hmm is there a way to retrieve the actual name of the ENV. would help this error message quite a lot.